Patch for issue #98

qwe661234 opened the pull request:
These are the patch commits for commit a304446.

qwe661234 committed:
Fix miss value fencei instrucition in branch list

In the previous implementation, fencei was treated as a branch
instruction, but it was assigned a missing value in the new branch list.
As a result, emulator fails to pass Zifencei test.

Close: #98

jserv reviewed (changes requested):
Refine the git commit messages. Ensure both the subject and body contain insightful materials.

Append `Close #98` at the end of git commit message.

jserv commented on src/emulate.c:
```diff
@@ -266,9 +266,8 @@ enum {
         rv->X[rv_reg_zero] = 0;                                           \
         code;                                                             \
+        rv->csr_cycle++;                                                  \
         if (__rv_insn_##inst##_canbranch) {                               \
```
Add comment back.

qwe661234 committed:
Fix missing csr_cycle increment

After finishing emulating, all instructions must increase csr cycle,
but we only do so for branch instructions.
